### ISSUE TITLE
Move debug logs to new file

### ISSUE THREAD

**Issue by jewzaam** [lightblue-platform/lightblue-mongo#112]
So it can be indexed differently than server.log, or not indexed at all.  This would allow us to log anything in the request and response.  It's not going to be useful in a managed environment unless it is indexed, though, so we'll have to work through how best to do that.

I was going to remove a bunch of debug statements from MongoCRUDController but there really seems to be value with those.

Maybe we could have a separate location for these that is controlled by puppet.  Then in dev we can dump them to server.log so they get sucked into splunk.  But in non-dev we have it go to /dev/null?

**Comment by bserdar** [comment]
Yes pls don't remove debug level logs. They are valueble for debugging, as
the name implies.
